lib/import/import_results.rb

```diff
@@ -0,0 +1,9 @@
+module Import
+  class ImportResults < Struct.new(:errors, :warnings, :successful_imports, :skipped_imports)
```

Comment:
Generally you shouldn't extend a struct. Can your ImportResults hold the struct instead. https://github.com/bbatsov/ruby-style-guide#no-extend-struct-new